snippets/post_install_network_config

```diff
@@ -77,6 +77,9 @@ mv /etc/sysconfig/network.cobbler /etc/sysconfig/network
 # Also set the hostname now, some applications require it
 # (e.g.: if we're connecting to Puppet before a reboot).
 /bin/hostname $hostname
+        #if $osversion == "rhel7"
```

Comment:
Tested once more. Even 'hostnamectl --static set-hostname $hostname' does not work in %post section. Always end up with localhost.localdomain.

Means to me either the 'hard way' (echo $hostname > /etc/hostname) in post_install_network_config snippet and **%post** section

or

follow the RedHat solution and put code to **%post --nochroot**.